cypress/integration/recieve-notice/recieve-notice-checkout.spec.js

```diff
@@ -0,0 +1,154 @@
+  const patronGroup = {};
+  const userData = { ...DefaultUser.defaultApiPatron };
+  const searchResultsData = {
+    userBarcode: userData.barcode,
+    itemBarcode: ITEM_BARCODE,
+    objectType: NOTICE_CATEGORIES.loan.name,
+    souce: 'ADMINISTRATOR, DIKU',
+    desc: 'Checked out to proxy: no.',
+    circAction: 'Checked out',
+  };
+  const testData = {};
+
+  beforeEach('Preconditions', () => {
+    noticePolicyTemplate.token = 'item.title';
+    noticePolicy.templateId = noticePolicyTemplate.name;
+    noticePolicy.format = 'Email';
+    noticePolicy.action = NOTICE_ACTIONS.checkout;
+    noticePolicy.noticeName = NOTICE_CATEGORIES.loan.name;
+    noticePolicy.noticeId = NOTICE_CATEGORIES.loan.id;
+    cy.getAdminToken()
+      .then(() => PatronGroups.createViaApi())
+      .then(res => {
+        cy.createUserApi({
+          patronGroup: res.id,
+          ...userData
+        }).then((createdUser) => {
+          userData.id = createdUser.id;
+        });
+        patronGroup.name = res.group;
+        patronGroup.id = res.id;
+      })
+      .then(() => {
+        cy.getServicePointsApi({ limit: 1, query: 'pickupLocation=="true"' }).then((res) => {
+          cy.addServicePointToUser(res[0].id, userData.id).then((points) => {
+            testData.userServicePoint = points.body.defaultServicePointId;
+          });
+        });
+      })
+      .then(() => {
+        cy.getMaterialTypes({ limit: 1 }).then((res) => { testData.materialType = res.id; });
+        cy.getLocations({ limit: 1 }).then((res) => { testData.location = res.id; });
+        cy.getHoldingTypes({ limit: 1 }).then((res) => { testData.holdingType = res[0].id; });
+        cy.getHoldingSources({ limit: 1 }).then((res) => { testData.holdingSource = res[0].id; });
+        cy.getInstanceTypes({ limit: 1 }).then((res) => { testData.instanceType = res[0].id; });
+        cy.getLoanTypes({ limit: 1 }).then((res) => { testData.loanType = res[0].id; });
+      })
+      .then(() => {
+        cy.createInstance({
+          instance: {
+            instanceTypeId: testData.instanceType,
+            title: `Pre-checkout_instance_${Number(new Date())}`,
+          },
+          holdings: [{
+            holdingsTypeId: testData.holdingType,
+            permanentLocationId: testData.location,
+            sourceId: testData.holdingSource,
+          }],
+          items: [
+            [{
+              barcode: ITEM_BARCODE,
+              missingPieces: '3',
+              numberOfMissingPieces: '3',
+              status: { name: 'Available' },
+              permanentLoanType: { id: testData.loanType },
+              materialType: { id: testData.materialType },
+            }],
+          ],
+        });
+      });
+    cy.loginAsAdmin({ path: settingsMenu.circulationPatronNoticePoliciesPath, waiter: NewNoticePolicyTemplate.waitLoading });
+  });
+
+  afterEach('Deleting created entities', () => {
+    cy.visit(settingsMenu.circulationPatronNoticePoliciesPath);
```

Comment:
move to the bottom to first steps in ui

Reply:
to the bottom of test body?

Reply:
I think just before NewNoticePolicyTemplate.waitLoading();